.storybook/metamask-storybook-theme.js

```diff
@@ -2,5 +2,5 @@
 export const metamaskStorybookTheme = {
   brandTitle: 'MetaMask Storybook',
   // Typography
-  fontBase: 'var(--font-family-sans)',
+  fontBase: 'var(--font-family-default)', // from @metamask/design-tokens stylesheet
```

Comment:
Replacing removed `--font-family-sans` with `--font-family-default` as per the [migration guide](https://github.com/MetaMask/metamask-design-system/blob/main/packages/design-tokens/MIGRATION.md#from-version-600-to-700)